sdk_core/CMakeLists.txt

```diff
@@ -42,26 +42,42 @@ target_compile_options(${SDK_LIBRARY_SHARED}
 
 set(LIVOX_PRIVATE_INCLUDE_DIR 
         ../3rdparty 
-        ../3rdparty/spdlog 
         .
         )
 set(LIVOX_PUBLIC_INCLUDE_DIR 
         ../include
         )
 
+find_package(RapidJSON REQUIRED)
+
 target_include_directories(
         ${SDK_LIBRARY_STATIC}
         PUBLIC
         ${LIVOX_PUBLIC_INCLUDE_DIR}
         PRIVATE
         ${LIVOX_PRIVATE_INCLUDE_DIR}
+        ${RapidJSON_INCLUDE_DIRS}
         )        
 target_include_directories(
         ${SDK_LIBRARY_SHARED}
         PUBLIC
         ${LIVOX_PUBLIC_INCLUDE_DIR}
         PRIVATE
         ${LIVOX_PRIVATE_INCLUDE_DIR}
+	${RapidJSON_INCLUDE_DIRS}
+        )
+
+find_package(spdlog REQUIRED)
+	
+target_link_libraries(
+        ${SDK_LIBRARY_STATIC}
+        PRIVATE
+	spdlog::spdlog
+        )
+target_link_libraries(
+        ${SDK_LIBRARY_SHARED}
+        PRIVATE
+	spdlog::spdlog
         )
```

Comment:
Are thesw two different libraries?

Reply:
Yes, they are the state lib (.a) and dynamic lib (.so) respectively. But we will only use the static lib.